### PR TITLE
Med 73 implement patient list UI

### PR DESCRIPTION
**Pull request by Mati-Pereira**
# UI for Patient List

patientlist
text

# features

- Ui for patient list

**PR checklist:**

**1.1 Common**
- [x] 1.1.2 no any, should be also added to eslint rules
- [x] 1.1.1 types for input and output params
- [x] 1.1.3 try/catch for any async function
- [x] 1.1.4 no magic numbers
- [x] 1.1.5 compare only with constants not with strings (instead of user === ‘admin’, better user === roles.admin)
- [x] 1.1.6 no ternary operator inside ternary operator (bad example: question ? first: second ? third: fourth)
- [x] 1.1.7 avoid similar types with duplicating by generics
- [x] 1.1.8 no console.log in pr
- [x] 1.1.9 .env file should be in .gitignore
- [x] 1.1.10 delete commented code if it's not part of documentation
- [x] 1.1.11 no links in the code, env links should be in env file (for example: server url),
- [x] constant links (for example default avatar URL) should be in constant file
- [x] 1.1.12 constants and function names should be lowercase.
- [x] 1.1.13 no dates format in the code (like "dd/MM/yyyy”), move it in global constant variable
- [x] 1.1.14 import rules. imports should come in a specific order: node modules, absolute path, relative path
- [x] 1.1.15 strings should be in the constant variable. Example: instead of ‘15 3 * * *’, const EACH_DAY_15h_15min = ‘15 3 * * *’

 **1.3 Frontend|Mobile**
- [x] 1.3.1 split component and logic (move logic to custom hook file)
- [x] 1.3.2 colors, font size, and font name should be in the theme or in the constants
- [x] 1.3.3 no text in the components, use i18n approach, even if you have only one language for now
- [x] 1.3.4 inline styles prohibited
- [x] 1.3.5 import should be absolute. instead of ../../../components/myComponent should be components/myComponent

### CODE REVIEW

**Review by oleksii-yurchenko — APPROVED** [review]

File: apps/medon-fe/src/app/app.tsx [hunk 1]
```diff
@@ -6,6 +6,9 @@ import ResetPassword from 'pages/ResetPassword';
 import ResendConfirmation from 'pages/ResendConfirmation';
 import UpdatePassword from 'pages/UpdatePassword';
 import Navigation from 'components/Navigation/index';
+import { PatientsPage } from 'pages/PatientsPage';
+import PatientsList from 'components/PatientsList';
+import { NewPatientForm } from 'components/NewPatientForm';
```
Comment (lyudad) [inline apps/medon-fe/src/app/app.tsx:11]: import ResendConfirmation from 'pages/ResendConfirmation';
import UpdatePassword from 'pages/UpdatePassword';
import { PatientsPage } from 'pages/PatientsPage';

import Navigation from 'components/Navigation/index';
import PatientsList from 'components/PatientsList';
import { NewPatientForm } from 'components/NewPatientForm';

File: apps/medon-fe/src/app/app.tsx [hunk 2]
```diff
@@ -18,6 +21,10 @@ function App() {
       <Route path="/reset-password/:token" element={<ResetPassword />} />
       <Route path="/re-confirm-account" element={<ResendConfirmation />} />
       <Route path="/update-password" element={<UpdatePassword />} />
+      <Route path="/patients" element={<PatientsPage />}>
```
Comment (lyudad) [inline apps/medon-fe/src/app/app.tsx]: all routes to constant file

File: apps/medon-fe/src/components/PatientsList/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import PatientListCard from 'components/PatientListCard';
+
+import { patientList } from 'utils/mock/patientList';
+import { ReactComponent as Plus } from 'assets/svgs/plus_listcard.svg';
+
+import { Choose, Content, StyledSearch, Wrapper } from './styles';
+
+export default function PatientsList() {
+  const theme = useTheme();
+  const { t } = useTranslation();
```
Comment (lyudad) [inline apps/medon-fe/src/components/PatientsList/index.tsx:14]: space line should be here, check your pritier configuration and rules

File: apps/medon-fe/src/components/PatientsList/index.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+          bgcolor={theme.colors.btnGradient}
+          to="add-new"
+        >
+          {t('patient-list.link')}
+          <Plus />
+        </LinkHome>
+      </Choose>
+      <h2>{t('patient-list.list')}</h2>
+      <Wrapper>
+        {patientList.map((patient) => (
```
Comment (lyudad) [inline apps/medon-fe/src/components/PatientsList/index.tsx]: what if patientList will be empty

**Pull request merged**